README.md

```diff
@@ -7,8 +7,7 @@
 [![NPM downloads weekly](https://badgen.net/npm/dw/highlight.js?label=npm+downloads&color=purple)](https://www.npmjs.com/package/highlight.js)
 [![jsDelivr CDN downloads](https://badgen.net/jsdelivr/hits/gh/highlightjs/cdn-release?label=jsDelivr+CDN&color=purple)](https://www.jsdelivr.com/package/gh/highlightjs/cdn-release)
 
-<!-- [![build and CI status](https://badgen.net/github/checks/highlightjs/highlight.js?label=build)](https://github.com/highlightjs/highlight.js/actions?query=workflow%3A%22Node.js+CI%22) -->
-![build and CI status](https://badgen.net/github/checks/highlightjs/highlight.js/main?label=build)
+[![ci status](https://github.com/highlightjs/highlight.js/actions/workflows/tests.js.yml/badge.svg?event=push)](https://github.com/highlightjs/highlight.js/actions/workflows/tests.js.yml)
```

Comment:
Was there some issue with the badgen badge?  We use the same source for all the badges for consistency.

Reply:
I don't know: I coppied the one that github officially provides.

Reply:
I'm ok with adding the link, but lets go back to using the previous badge - then this would be mergable.

Thanks!